contracts/HashRegistrarSimplified.sol

```diff
@@ -130,8 +130,7 @@ contract Registrar {
     //   Auction -> Reveal
     //   Reveal -> Owned
     //   Reveal -> Open (if nobody bid)
-    //   Owned -> Forbidden (invalidateName)
-    //   Owned -> Open (releaseDeed)
+    //   Owned -> Open (releaseDeed or invalidateName)
```

Comment:
I left it in because I was concerned about exactly that con. We could rename it to something like 'OBSOLETE_FORBIDDEN'.